palisade-service/README.md

```diff
@@ -47,10 +53,9 @@ The application exposes two REST endpoints:
 
 ## Kafka Interface
 
-The application receives a REST request containing a `PalisadeRequest`, and after returning a token to the client, creates the information we need for Kafka, including a Start and End of Stream message, containing the relevant headers. It then creates a
-message in-between the start and end, containing the
-`PalisadeRequest` object, which contains the user, resourceId and context. The `x-request-token` is sent in the Kafka headers. In case of errors, the original request and thrown exception are both captured in an `AuditErrorMessage` and written to the
-Kafka `error` topic.
+The application receives a REST request containing a `PalisadeRequest`, and after returning a token to the client, creates the information we need for Kafka, including a Start and End of Stream message, containing the relevant headers. 
+It then creates a message in-between the start and end, containing the `PalisadeRequest` object, which contains the user, resourceId and context. 
```

Comment:
PalisadeSystemResponse instead of PalisadeRequest.

Reply:
updated